Source/LoadByOS/WindowsConfigApp/WinInterop.cs

```diff
@@ -36,16 +32,17 @@ public void SetFolderIcon(string image, string directory)
 
         public void DeleteFolderIcon(string directory)
             => new DirectoryInfo(directory)?.DeleteIcon();
-        public bool CanUpdate => true;
-		public void InstallUpdate(string updateBundle)
+
+        public bool CanUpgrade => true;
+		public void InstallUpgrade(string upgradeBundle)
 		{
 			var thisExe = Environment.ProcessPath;
 			var thisDir = Path.GetDirectoryName(thisExe);
 			var zipExtractor = Path.Combine(Path.GetTempPath(), "ZipExtractor.exe");
 
 			File.Copy("ZipExtractor.exe", zipExtractor, overwrite: true);
 
-			RunAsRoot(zipExtractor, $"--input \"{updateBundle}\" --output \"{thisDir}\" --executable \"{thisExe}\"");
+			RunAsRoot(zipExtractor, $"--input \"{upgradeBundle}\" --output \"{thisDir}\" --executable \"{thisExe}\"");
```

Comment:
In case it's useful (I couldn't figure out your spiffy embedding trick):

https://github.com/rmcrackan/Dinah.Core/blob/master/Dinah.Core/StringExtensions.cs#L42

Reply:
Thanks, I'll use that. Unfortunately, embedded code only works in comments on the repository you're linking to.